src/OpenTK/Platform/Windows/WinGLNative.cs

```diff
@@ -735,7 +735,7 @@ private IntPtr WindowProcedure(IntPtr handle, WindowMessage message, IntPtr wPar
                     break;
 
                 case WindowMessage.ERASEBKGND:
-                    return new IntPtr(1);
+                    break;
```

Comment:
I'll double-check on this, but I recall it only reaching this case once during execution.